src/conn/conn_compact.c

```diff
@@ -18,13 +18,198 @@ __compact_server_run_chk(WT_SESSION_IMPL *session)
     return (FLD_ISSET(S2C(session)->server_flags, WT_CONN_SERVER_COMPACT));
 }
 
+/*
+ * __get_compact_stat --
+ *     Get the statistics for the given uri.
+ */
+static WT_BACKGROUND_COMPACT_STAT *
+__get_compact_stat(WT_SESSION_IMPL *session, const char *uri)
```

Comment:
Thanks for pointing that out!

Reply:
@agorrod, should we have the prefix `background` or `bg` for the functions?
For instance, given the example below, which one should we go for?
```
__compact_server_run_chk

1) __background_compact_server_run_chk
2) __bg_compact_server_run_chk
```

The first option is quite verbose, the second has an acronym that we try to avoid.

Reply:
I'd go with `background`. I prefer only to introduce shortenings for *very* commonly used concepts to keep the number I need to remember manageable, and I don't think this qualifies.
This leans into personal preference, which is why I'm giving a reason for my preference.

Reply:
Thanks for the input, I'll keep `__background_compact` as the prefix.